include/drivers/dma.h

```diff
@@ -98,25 +99,26 @@ struct dma_block_config {
 /**
  * @brief DMA configuration structure.
  *
- *     dma_slot             [ 0 : 5 ]   - which peripheral and direction
+ *     dma_slot             [ 0 : 6 ]   - which peripheral and direction
  *                                        (HW specific)
- *     channel_direction    [ 6 : 8 ]   - 000-memory to memory, 001-memory to
+ *     channel_direction    [ 7 : 9 ]   - 000-memory to memory, 001-memory to
```

Comment:
Nit: let's use 1 value per line in field comment

Reply:
ok